Geometry/VeryForwardGeometryBuilder/src/DetGeomDesc.cc

```diff
@@ -152,9 +152,7 @@ DiamondDimensions DetGeomDesc::computeDiamondDimensions(const bool isABox,
       boxShapeParameters = {params.at(0), params.at(1), params.at(2)};
     } else {
       // convert cm (DD4hep) to mm (legacy expected by PPS reco software)
```

Comment:
Comments on lines 50, 57, and 154 need to be updated. DD4hep units are not necessarily cm anymore.
```
 // Convert DD4hep units to mm (legacy expected by PPS reco software)
```

Reply:
@cvuosalo Same, sorry will quickly address the comment tonight when I am back to a computer